### PR TITLE
Add blog-post route information including ember-data magic

### PR DESCRIPTION
**Pull request by hbrysiewicz**

### CODE REVIEW

File: outline.md [hunk 1]
```diff
@@ -292,23 +292,15 @@ Instead let's use the data store to retrieve all of our blog posts:
...
-TODO: explain what it looks like when it worked and add a screenshot
-
-#### Acceptance testing
-
-With some user interaction added to our application we can now create an acceptance test.
-
-Let's test that the blog posts from our API show up on the homepage.
-
-TODO: Add acceptance test for our blog post models showing up
+![display all blog posts](https://s3.amazonaws.com/f.cl.ly/items/0e0w0u2b2o3d0r47301y/Screen%20Shot%202015-03-31%20at%201.45.32%20PM.png)
```
Comment (treyhunner) [inline outline.md:303]: How did you make that URL? Should we store screenshots in the same directory under git?

Comment (hbrysiewicz) [inline outline.md:303]: My screenshot app Cloudly auto-uploads it for me to share. If you want to download and store it within the repo idm

File: outline.md [hunk 2]
```diff
@@ -343,34 +335,164 @@ This creates a few files, and also adds a  `app/router.js`:
...
 
     <article>
       <h2>{{model.title}}</h2>
       {{model.body}}
     </article>
 
-If we visit `http://localhost:4200/post/1` in our browser, we should see an example blog post.
+Since we happen to know there is a blog post with `id: 1` on our API server, we can manually visit `http://localhost:4200/post/1` in our browser to test with an example blog post.
+
+#### The magic of Ember-Data
```
Comment (treyhunner) [inline outline.md:360]: Should this section replace the API table/explanation in the section [8 Diversion: Accessing our API with ember-data](https://github.com/hbrysiewicz/ember-cli-101/blob/master/outline.md#8-diversion-accessing-our-api-with-ember-data)?

**Comment by treyhunner** [comment]
:+1: looks good. I think there might be a little duplication with one of the current sections that should probably be removed or modified.

File: outline.md (hunk 2 above)
Comment (hbrysiewicz) [inline outline.md:360]: Probably not because that refers to the RAILS API

Comment (treyhunner) [inline outline.md:360]: That makes sense

**Pull request merged**